packages/base/adapters/evm/ethers/index.ts

```diff
@@ -1,6 +1,7 @@
 import '@web3modal/polyfills'
 
 export { EVMEthersClient } from './client.js'
+export * from '@web3modal/scaffold-utils/ethers'
```

Comment:
is this needed?

Reply:
Yes, we are connecting to the `EthersStoreUtil` from the backwards compat packages' custom hooks, [see](https://github.com/WalletConnect/web3modal/blob/main/packages/ethers/exports/react.tsx#L46). To make the `ethers` and `base` package use the same proxy object (state), the `@web3modal/scaffold-utils/ethers` needed to be exported from the adapter. Otherwise throwing this https://github.com/pmndrs/valtio/discussions/706 error.